docs/os/windows/windows-vm41.md

```diff
@@ -0,0 +1,203 @@
+Installing a Windows VM
+=======================
+
+Simple Windows install
+----------------------
+
+If you just want something simple and you can live without some features. This works for Windows XP, 7, 10 and 11, and it may work for Windows 8 and 8.1, although this has not been tested.
```

Comment:
I don't think the sentence "If you just want something simple and you can live without some features" is really clear (I know it isn't yours). Maybe remove it ? Also, the works/doesn't work features list below don't depend on the type of installation (ie. GUI or CLI) and should maybe move to a standalone section, with a note about QWT (I'm trying to read the instructions as a new user - at that point I'm not even aware of QWT and how it would "fix" the "doesn't work" features ! - so maybe add a bit of text about it + a link to the new tools41 doc ?).

Reply:
I restructured it a bit and removed the old "simple install", which I didn't like anyhow. I just left the old short CLI commands list, as such lists are used in other parts of the Qubes documentation. In the detailed instructions I tried to cover both CLI and GUI usage.